cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -446,6 +461,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             var is_proctored_exam;
             var time_limit = this.getExamTimeLimit();
             var exam_review_rules = this.$('#id_exam_review_rules').val();
+            var hide_after_due = this.$('#id_hide_exam_after_due_date').is(':checked');
```

Comment:
Note that since you always send this checked value (regardless of whether or not it is a timed exam), you may end up sending "true" for a proctored exam if the author selected "hide after due" while Timed exam was selected. This is probably fine since it is just ignored by proctored exams, but think through if there are any issues with this.